lib/generators/suspenders/rake_generator.rb

```diff
@@ -0,0 +1,25 @@
+module Suspenders
+  module Generators
+    class RakeGenerator < Rails::Generators::Base
+      source_root File.expand_path("../../templates/rake", __FILE__)
+      desc(<<~TEXT)
+        Configures the default Rake task to audit and lint the codebase with `bundler-audit` and `standard`.
+      TEXT
+
+      def add_bundler_audit_and_standard
+        gem_group :development, :test do
+          gem "bundler-audit", ">= 0.7.0", require: false
+          gem "standard"
+        end
+        Bundler.with_unbundled_env { run "bundle install" }
+      end
+
+      def configurea_rake_task
+        copy_file "bundler_audit.rake", "lib/tasks/bundler_audit.rake"
+
+        insert_into_file "Rakefile", "\nrequire \"standard/rake\"", after: "require_relative \"config/application\""
+        append_file "Rakefile", %(\ntask default: %i[bundle:audit standard]\n)
```

Comment:
Not sure if it's worth accounting for RSpec like we [currently](https://github.com/thoughtbot/suspenders/blob/55cb5c246fe53aea4cf53436c7c208b40ad13e85/lib/suspenders/app_builder.rb#L231) do?

Reply:
We don't need to include RSpec in the Rakefile. I tested with a simple app using RSpec, and by default, running `bundle exec rake` runs RSpec. This was still true after having invoked `suspenders:rake`, except that `standard` and `bundler-audit` were also invoked.